src/operation/decision_model.jl

```diff
@@ -135,7 +135,7 @@ function DecisionModel{M}(
 end
 
 """
-This builds the optimization problem of type M with the specific system and template
+Builds the optimization problem of type M with the specific system and template
```

Comment:
Per Julia rules it should be `Build the optimization …`
https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation